PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/collectioninterfaces/IGenomeIterable.java

```diff
@@ -0,0 +1,23 @@
+//package nl.tudelft.pl2016gr2.collectioninterfaces;
```

Comment:
Why is the code in this class commented? Is it still going to be used? If not, it should be deleted.